src/lib/swapper/swappers/CowSwapper/utils/constants.ts

```diff
@@ -20,3 +22,8 @@ export const ERC20_TOKEN_BALANCE = 'erc20'
 // Address used by CowSwap to buy ETH
 // See https://github.com/gnosis/gp-v2-contracts/commit/821b5a8da213297b0f7f1d8b17c893c5627020af#diff-12bbbe13cd5cf42d639e34a39d8795021ba40d3ee1e1a8282df652eb161a11d6R13
 export const COW_SWAP_NATIVE_ASSET_MARKER_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
+
+export const COW_SWAP_SUPPORTED_CHAIN_IDS: SupportedChainIds = {
+  sell: getSupportedChainIds(),
+  buy: getSupportedChainIds(),
```

Comment:
While at it, can we use this PR (or do it as an atomic follow-up, either works) to remove this method/ Gnosis flag and move everything to this constant?
The only reason we had it was because of `isGnosisEnabled` to conditionally support Gnosis, and it's been stable for a long long time now

Reply:
easy, here ya go 5ff8a6ae6e8103384495eebfa3960ba723df0f27